include/nil/blueprint/assigner.hpp

```diff
@@ -1875,6 +1821,7 @@ namespace nil {
             generation_mode gen_mode;
             std::vector<branch_desc> curr_branch;
             component_calls statistics;
+            std::vector<typename BlueprintFieldType::value_type> internal_storage;
```

Comment:
Also probably it makes sense to alias the used type. Now you have to know the semantics of this vector (how its indices and values are interpreted). Probably it would be nice to have e.g. `internal_storage = std::vector<typename BlueprintFieldType::value_type>` (or maybe there is some better naming) type with documented interpretation of indices and values.

Reply:
will be `assignment<ArithmetizationType>::ColumnType internal_storage` which is `std::vector<typename FieldType::value_type>`